### ISSUE TITLE
IMPORT ERROR

### ISSUE THREAD

**Issue by Daftscientist** [ComatoseCow/Moosician#1]
hi, i get this error when running the bot. im not sure how to fix?
`Traceback (most recent call last):
  File "c:/Users/LeoJo/Desktop/music bot test/bot.py", line 5, in <module>
    import utils._config as Config # pylint: disable=import-error
ModuleNotFoundError: No module named 'utils._config'`

**Comment by TrissieTheHusky** [comment]
Have you changed the config files like it says to?

**Comment by TrissieTheHusky** [comment]
It looks like what it's saying is the file in your **utils/** folder is not present did you delete it?

**Comment by ComatoseCow** [comment]
As Trissie said it looks like it is not finding the utils folder or the _config.py in that folder.

**Comment by Daftscientist** [comment]
Yes it is there all edited names. Should I os define the directory?

**Comment by ComatoseCow** [comment]
What os are you running on?
The error seems to be an issue with the working directory is my guess.
The code was created and is run on a windows 10 machine for me and my friends discord servers so folder structure may e an issue.
